### DISCUSSION TITLE
Is the ftest used in combined_ftest_5x2cv the same as the F-test

### DISCUSSION THREAD

**Discussion by stephano41** [rasbt/mlxtend#999] [Q&A] [answered]
Just wondering if the ftest used in the function [combined_ftest_5x2cv](http://rasbt.github.io/mlxtend/user_guide/evaluate/combined_ftest_5x2cv/) is the same as the one used in [F-test](http://rasbt.github.io/mlxtend/user_guide/evaluate/ftest/)?

Am hoping to test multiple models by running each with a 5x2 cv first, then use a cochrane Q test to compare those results, then do a posthoc analysis of those results using pairwise f test as detailed in [F-test](http://rasbt.github.io/mlxtend/user_guide/evaluate/ftest/)

**Comment by rasbt** [comment, marked as answer]
Oh yeah, they are certainly closely related. Both use a F-distribution to compute the p-value. However, the score and degrees of freedom are computed differently due to the 5x2 procedure.
